app/react/reducers/index.js

```diff
@@ -0,0 +1,13 @@
+import { combineReducers } from 'redux';
+
+import MessagesReducer from './reducer_messages';
+import ActiveRoomReducer from './reducer_active_room';
+import RoomsReducer from './reducer_rooms';
+
+const rootReducer = combineReducers({
+  messages: MessagesReducer,
+  activeRoomId: ActiveRoomReducer,
+  rooms: RoomsReducer,
+});
```

Comment:
combining reducers is a common pattern, consider this syntax in order not to make it too verbose. Of course you'd need to rename the files.

```javascript
import { combineReducers } from 'redux';

import messages from './messages';
import activeRoomId from './active_room_id';
import rooms from './rooms';

const rootReducer = combineReducers({
  messages,
  activeRoomId,
  rooms,
});

export default rootReducer;
```